PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+TO CONTRIBUTORS:
+
+Make sure you have:
+
+ * [ ] reviewed and applied the coding style: [coding](./docs/style.md), [naming](./docs/naming.md)
+ * [ ] for tactics:
+     * [ ] added or adapted documentation in [tactics.md](./docs/tactics.md)
+     * [ ] write an example of use of the new feature in [tactics.lean](./tests/tactics.lean)
+  * [ ] reviewed other PRs to avoid redundancy or overlap
+
+TO REVIEWERS:
+
+Check:
+
+ * [ ] coding style
+ * [ ] for tactics:
+     * [ ] documentation
+     * [ ] tests
+     * [ ] efficiency (make sure at least it's not outrageously inefficient)
+  * [ ] it fits the overall mathlib design
```

Comment:
I don't think the reviewer checklist should be here - this list is only modifiable by the original author. It can go in a wiki page as general reference, but it doesn't seem like it would be useful here.

Reply:
The reason I'm putting it here is that github labels PRs with the number of boxes left unticked. If different people check different parts of a PR, this should help tell you at a glance what is left to review.

Reply:
done